uniflow/flow/config.py

```diff
@@ -221,22 +221,22 @@ class RaterForClassificationOpenAIGPT4Config(RaterConfig):
             instruction="""
             Evaluate the appropriateness of a given answer based on the question and the context.
             There are few examples below, consisting of context, question, answer, explanation and label.
-            If answer is appropriate, you should give a higher score and vise versa. Check label to score dictionary: {label2score}.
+            If answer is appropriate, you should give a label representing higher score and vise versa. Check label to score dictionary: {label2score}.
             Your response should only focus on the unlabeled sample, including two fields: explanation and label (one of {label_list}).
             """,
             few_shot_prompt=[
                 Context(
                     context="The Eiffel Tower, located in Paris, France, is one of the most famous landmarks in the world. It was constructed in 1889 and stands at a height of 324 meters.",
                     question="When was the Eiffel Tower constructed?",
                     answer="The Eiffel Tower was constructed in 1889.",
-                    explanation="The context explicitly mentions that the Eiffel Tower was constructed in 1889, so the answer is correct.",
+                    explanation="The context explicitly mentions that the Eiffel Tower was constructed in 1889, so the answer is correct and asigned a score of 1.0.",
```

Comment:
Is there a reason why added this as a part of the CoT explanation?

In this [line](https://github.com/CambioML/uniflow/pull/97/files#diff-1d136208e1ea8e3149fbce3118d24262814be2c6e5a1db58a48b739e8be13701R151-R152), you past in both `label_list` as well as `label2score` both as prompt input. However, this is a bit over-engineered. You should only need language model to reason to give Yes or No. Then, after you have the answer, you can use labe2score dict to map it to the corresponding value. You explanation can cause hallucination because `-1` is not a value in label2score dict and it is not necessary.

Reply:
sorry `-1` is a typo... this is because originally I want to keep origin prompt is valid when users want to change `label2score`, they do not have to change `instruction` in `prompt_template`. However, `examples` in `prompt_template` must be changed. Can I just keep instruction and delete "and asigned a score of 1.0"?